frontend/src/components/design-system/containers/CardComponent.vue

```diff
@@ -0,0 +1,91 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+
+export interface Props {
+  radius?: 'small' | 'large';
+  corner?: 'top-left' | 'top-right' | 'bottom-left' | 'bottom-right';
+  border?: boolean | 'muted';
+  shadow?: boolean;
+  padding?: 'small' | 'large';
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  radius: 'small',
+  corner: 'top-left',
+  border: true,
+  shadow: false,
+  padding: 'small',
+});
+
+const classes = computed(() => ({
+  [`radius-${props.radius}`]: true,
+  [`corner-${props.corner}`]: true,
+  border: props.border,
+  'border-muted': props.border === 'muted',
+  shadow: props.shadow,
+  [`padding-${props.padding}`]: true,
+}));
+</script>
+
+<template>
+  <div :class="classes">
+    <slot />
+  </div>
+</template>
+
+<style scoped lang="scss">
+div {
+  background-color: rgb(var(--color-neutral-bg-container));
+}
+
+.radius {
+  &-small {
+    border-radius: var(--radius-small);
+  }
+  &-large {
+    border-radius: var(--radius-large);
+  }
+}
+
+.corner {
+  &-top-left {
```

Comment:
ja vim ze sme se na BEM neshodli ale je to dobrej standard :) corner--top-left je citelnejsi nez corner-top-left. Je alespon videt co je tam ta nadrazena class. Kazdopadne tohle je jenom suggestion.

Reply:
Upraveno na `--`.